frontend/src/app/main/ui/workspace/context_menu.cljs

```diff
@@ -64,22 +90,33 @@
              (when (and (some? dom) (some? submenu-node))
                (dom/set-css-property! submenu-node "top" (str (.-offsetTop dom) "px"))))))]
 
-    [:li {:ref set-dom-node
-          :on-click on-click
-          :on-pointer-enter on-pointer-enter
-          :on-pointer-leave on-pointer-leave}
-     [:span.title title]
-     [:span.shortcut (or shortcut "")]
-
-     (when (> (count children) 1)
-       [:span.submenu-icon i/arrow-slide])
-
-     (when (> (count children) 1)
-       [:ul.workspace-context-menu
-        {:ref submenu-ref
-         :style {:display "none" :left 250}
-         :on-context-menu prevent-default}
-        children])]))
+    (if has-icon?
+      [:li.sub-menu-item {:ref set-dom-node
```

Comment:
Perhaps is better to name this class .icon-menu-item? Because I think it does not apply to all sub menus, but only those with icons.

Reply:
For sure¡¡